File: src/main/java/org/shark/example/service/cassandra/GetAccountListService.java
package org.shark.example.service.cassandra;

import lombok.RequiredArgsConstructor;
import org.shark.example.datasource.cassandra.AccountRepository;
import org.shark.example.datasource.cassandra.pojo.AccountDo;
import org.springframework.stereotype.Service;

import java.util.List;

@RequiredArgsConstructor
@Service
public class GetAccountListService {

    private final AccountRepository accountRepository;

    public List<AccountDo> getAccountList() {
        return accountRepository.findAll();
    }

}

File: src/main/kotlin/org/shark/example/service/cassandra/GetAccountListService.kt
package org.shark.example.service.cassandra

import org.shark.example.datasource.cassandra.AccountRepository
import org.shark.example.datasource.cassandra.pojo.AccountDo
import org.springframework.stereotype.Service

@Service
class GetAccountListService(
    private val accountRepository: AccountRepository
) {
    fun getAccountList(): List<AccountDo> {
        return accountRepository.findAll()
    }
}
